Magento_PageBuilder/styles/modules/content-type/_import.scss

```diff
@@ -18,3 +18,6 @@
 @import 'tabs';
 @import 'text';
 @import 'video';
+
+// styles adjustments
+@import 'adjustments';
```

Comment:
maybe we can name it `page-builder-adjustments` or `pb-adjustments`?

Reply:
All imports has simple names and they are in `Magento_PageBuilder` folder so imo it's not necessary. But if You think we should change it I think we should change all of them not only "adjustments".

Reply:
ok, let's leave it, it's indeed in `Magento_PageBuilder` so, it's rather clear